cg/services/post_processing/exc.py

```diff
@@ -0,0 +1,5 @@
+from cg.exc import CgError
```

Comment:
We could refine if we want a parent post-processing error and many errors inheriting from it, but for now it looks good